controller/volume_controller.go

```diff
@@ -2600,12 +2604,12 @@ func (c *VolumeController) getNodeCandidatesForAutoBalanceZone(v *longhorn.Volum
 		return candidateNames
 	}
 
-	readyNodes, err := c.ds.ListReadyAndSchedulableNodes()
+	readyNodes, err := c.ds.ListReadyAndSchedulableNodesRO()
```

Comment:
What's the difference between `ListReadyAndSchedulableNodesRO` and `ListReadyAndSchedulableNodes` for `delete(readyNodes, nName)`?

Reply:
Here, the operations like `delete(readyNodes, nName)` do not include object updating over `readyNodes`. Hence, there is no need to do deep copy during the node list

Reply:
Got it. Thank you.